elastic/synthetics-recorder
# feat: windows sign endpoint to http://timestamp.digicert.com

## Summary

http://timestamp.acs.microsoft.com was down so let's use http://timestamp.digicert.com. That endpoint has been used in the Unified Release process for quite sometime and it's stable

Interestingly the metadata file shows the other azure endpoint though things worked fine

<img width="618" alt="image" src="https://github.com/elastic/synthetics-recorder/assets/2871786/49f671b2-ff2f-4273-91b1-68d2b6a4c680">






## Implementation details





## How to validate this change

In the CI, see this [build](https://internal-ci.elastic.co/job/elastic+synthetics-recorder+release/view/tags/job/v1.3.0-rc8/)
